scripts/watch.cjs

```diff
@@ -153,19 +160,29 @@ const setupExtensionApiWatcher = name => {
 
 (async () => {
   try {
+    const extensions = []
+    for(let index=0; index < process.argv.length;index++) {
+      if (process.argv[index] === EXTENSION_OPTION && index < process.argv.length - 1) {
+        extensions.push(path.resolve(process.argv[++index]));
+      }
+    }
     const viteDevServer = await createServer({
       ...sharedConfig,
       configFile: 'packages/renderer/vite.config.js',
+      extensions: extensions
     });
 
     await viteDevServer.listen();
-    await setupExtensionApiWatcher('compose');
-    await setupExtensionApiWatcher('docker');
-    await setupExtensionApiWatcher('kube-context');
-    await setupExtensionApiWatcher('lima');
-    await setupExtensionApiWatcher('podman');
-    await setupExtensionApiWatcher('kind');
-    await setupExtensionApiWatcher('registries');
+    await setupExtensionApiWatcher(path.resolve(__dirname, '../extensions/compose'));
+    await setupExtensionApiWatcher(path.resolve(__dirname, '../extensions/docker'));
+    await setupExtensionApiWatcher(path.resolve(__dirname, '../extensions/kube-context'));
+    await setupExtensionApiWatcher(path.resolve(__dirname, '../extensions/lima'));
+    await setupExtensionApiWatcher(path.resolve(__dirname, '../extensions/podman'));
+    await setupExtensionApiWatcher(path.resolve(__dirname, '../extensions/kind'));
+    await setupExtensionApiWatcher(path.resolve(__dirname, '../extensions/registries'));
```

Comment:
I don't see why we would need that change

Reply:
I mean, why from something simple where we give the name of the extension, we're having `path.resolve(__dirname, '../extensions/compose')`

we should still have a function with only the name of the extension or you iterate from the extensions folder automatically but here it looks too verbose

Reply:
could be like `setupBuiltinExtensionApiWatcher('compose')` that calls `setupExtensionApiWatcher()` with `path.resolve(__dirname,`

Reply:
or: `setupBuilltinExtensionsApiWatcher()` that list all folders of /extensions and then call `setupExtensionApiWatcher()`with `path.resolve`